lib/diplomat/key_utils.ex

```diff
@@ -0,0 +1,200 @@
+defmodule KeyUtils do
```

Comment:
Urlsafe stuff could be use as an unique id for an entity. 
If an entity have not parent, we could use Key.id as unique id, but when entity has parent, 2 entities of same kind could have same id. In that case, I must use `urlsafe` as unique id

I am migrating our application from Python to Elixir, and in Python app, I already use `urlsafe` stuff, so I think it is helpful if the Elixir library also support implement this function.